docs/source/quicktour.md

```diff
@@ -68,83 +62,103 @@ model.print_trainable_parameters()
+<Tip>
 
-  model = model.to(device)
-  model.eval()
-  inputs = tokenizer("Tell me the recipe for chocolate chip cookie", return_tensors="pt")
+Take a look at the [AutoPeftModel](package_reference/auto_class) API reference for a complete list of available `AutoPeftModel` classes.
 
-  with torch.no_grad():
-      outputs = model.generate(input_ids=inputs["input_ids"].to("cuda"), max_new_tokens=10)
-      print(tokenizer.batch_decode(outputs.detach().cpu().numpy(), skip_special_tokens=True)[0])
-  'Tell me the recipe for chocolate chip cookie dough.
-  1. Preheat oven'
-```
+</Tip>
 
-## Easy loading with Auto classes 
+Easily load any PEFT-trained model for inference with the `AutoPeftModelFor` class using the using the [`~transformers.PreTrainedModel.from_pretrained`] method:
 
-If you have saved your adapter locally or on the Hub, you can leverage the `AutoPeftModelForxxx` classes and load any PEFT model with a single line of code:
+```py
+from peft import AutoPeftModelForCausalLM
+from transformers import AutoTokenizer
+import torch
 
-```diff
-- from peft import PeftConfig, PeftModel
-- from transformers import AutoModelForCausalLM
-+ from peft import AutoPeftModelForCausalLM
+model = AutoPeftModelForCausalLM.from_pretrained("ybelkada/opt-350m-lora")
+tokenizer = AutoTokenizer.from_pretrained("facebook/opt-350m")
 
-- peft_config = PeftConfig.from_pretrained("ybelkada/opt-350m-lora") 
-- base_model_path = peft_config.base_model_name_or_path
-- transformers_model = AutoModelForCausalLM.from_pretrained(base_model_path)
-- peft_model = PeftModel.from_pretrained(transformers_model, peft_config)
-+ peft_model = AutoPeftModelForCausalLM.from_pretrained("ybelkada/opt-350m-lora")
-```
+model = model.to("cuda")
+model.eval()
+inputs = tokenizer("Preheat the oven to 350 degrees and place the cookie dough", return_tensors="pt")
 
-Currently, supported auto classes are: `AutoPeftModelForCausalLM`, `AutoPeftModelForSequenceClassification`, `AutoPeftModelForSeq2SeqLM`, `AutoPeftModelForTokenClassification`, `AutoPeftModelForQuestionAnswering` and `AutoPeftModelForFeatureExtraction`. For other tasks (e.g. Whisper, StableDiffusion), you can load the model with:
+with torch.no_grad():
+    outputs = model.generate(input_ids=inputs["input_ids"].to("cuda"), max_new_tokens=50)
+    print(tokenizer.batch_decode(outputs.detach().cpu().numpy(), skip_special_tokens=True)[0])
+
+"Preheat the oven to 350 degrees and place the cookie dough in the center of the oven. In a large bowl, combine the flour, baking powder, baking soda, salt, and cinnamon. In a separate bowl, combine the egg yolks, sugar, and vanilla."
+```
 
-```diff
-- from peft import PeftModel, PeftConfig, AutoPeftModel
-+ from peft import AutoPeftModel
-- from transformers import WhisperForConditionalGeneration
+For other tasks that aren't explicitly supported with an `AutoPeftModelFor` class - such as automatic speech recognition - you can still use the base [`AutoPeftModel`] class.
```

Comment:
I think this can be something we put in the Tutorial section (this is good basic info for new users to learn since `AutoPeftModel` is just a wrapper around `PeftModel`)? For the quicktour though, I think we should aim for convenience to get users started as quickly as possible :)